play-frontend-hmrc-play-30/src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/servicenavigation/ServiceNavigationItem.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.govukfrontend.views.viewmodels
+package servicenavigation
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.{Content, Empty}
+
+case class ServiceNavigationItem(
+  content: Content = Empty,
+  href: String = "",
+  active: Boolean = false,
+  current: Boolean = false,
```

Comment:
so (from looking at govuk-frontend) current just means use `aria-current="page"` rather than `aria-current="true"` not going to be a confusing api to use at all... 💀

Reply:
Yep, this is full of gems like that 😅